dedibox-console/account/how-to/resolve-abuse.mdx

```diff
@@ -5,16 +5,16 @@ meta:
 content:
   h1: How to resolve Dedibox abuse tickets and cases
   paragraph: This page explains how to resolve Abuse tickets.
-tags: resolve abuse complaint ticket dedibox-console 
+tags: resolve abuse complaint ticket dedibox-console
 dates:
-  validation: 2024-02-19
+  validation: 2024-09-02
   posted: 2020-03-12
 categories:
   - account
 ---
 
 
-Abuses are complaints regarding the use of Scaleway services for illegal behavior or hosting of illegal content. Abuse reports for the same issue on the same resource within a given time frame are grouped by case.
+Abuse tickets are complaints regarding the use of Scaleway services for illegal behavior or hosting of illegal content. Abuse reports for the same issue on the same resource within a given time frame are grouped by case.
```

Comment:
Maybe it should be Abuse reports, seeing as that's what they're referred to as in the second sentence?

Reply:
```suggestion
Abuse reports are complaints regarding the use of Scaleway services for illegal behavior or hosting of illegal content. Abuse reports for the same issue on the same resource within a given time frame are grouped by case.
```